inkcpp/output.cpp

```diff
@@ -79,7 +114,7 @@ namespace ink
 			template<typename OUT>
 			inline void write_char(OUT& output, char c)
 			{
-				static_assert(false, "Invalid output type");
+				static_assert(! std::is_same<OUT,OUT>::value, "Invalid output type");
```

Comment:
The compiler is smart and therefor a `static_assert(false` always fails, even when it isn't instantiated, but when you put a template inside, it's need an instantiation to fail.